tracee-ebpf/tracee/net_events.go

```diff
@@ -28,85 +36,44 @@ func (t *Tracee) processNetEvents() {
 				continue
 			}
 
-			evtMeta, dataBuff := parseEventMetaData(in)
+			evtMeta, payloadBytes := parseEventMetaData(in)
 
-			// timeStamp is nanoseconds since system boot time
 			timeStampObj := time.Unix(0, int64(evtMeta.TimeStamp+t.bootTime))
 
+			if t.config.Output.RelativeTime {
+				// To get the current ("wall") time, we add the boot time into it.
+				evtMeta.TimeStamp -= t.startTime
+			} else {
+				// timeStamp is nanoseconds since system boot time
+				// To get the monotonic time since tracee was started, we have to subtract the start time from the timestamp.
+				evtMeta.TimeStamp += t.bootTime
+			}
+
 			if evtMeta.NetEventId == NetPacket {
-				var pktLen uint32
-				err := binary.Read(dataBuff, binary.LittleEndian, &pktLen)
-				if err != nil {
-					t.handleError(err)
-					continue
-				}
-				var ifindex uint32
-				err = binary.Read(dataBuff, binary.LittleEndian, &ifindex)
+				captureData, err := parseCaptureData(payloadBytes)
 				if err != nil {
 					t.handleError(err)
 					continue
 				}
-				idx, ok := t.ngIfacesIndex[int(ifindex)]
-				if !ok {
+				if err := t.writePacket(captureData, time.Unix(int64(evtMeta.TimeStamp), 0), bytes.NewBuffer(payloadBytes)); err != nil {
 					t.handleError(err)
 					continue
 				}
-
 				if t.config.Debug {
-					var pktMeta struct {
-						SrcIP    [16]byte
-						DestIP   [16]byte
-						SrcPort  uint16
-						DestPort uint16
-						Protocol uint8
-						_        [3]byte //padding
-					}
-					err = binary.Read(dataBuff, binary.LittleEndian, &pktMeta)
+					networkProcess, err := t.getProcessCtx(int(evtMeta.HostTid))
 					if err != nil {
 						t.handleError(err)
```

Comment:
Do we really want to treat this as an error in case process context wasn't found? It's different from the previous behavior were we just omit some fields in such a case.